docs/01_nodeos/07_concepts/15_private_chain_access/index.md

```diff
@@ -0,0 +1,79 @@
+---
+content_title: Private Chain Access Control
+link_text: Private Chain Access Control
+---
+
+## Overview
+
+The *Private Access Control* or *Security Group* feature provides a two-layer security solution for blockchain administrators to control which participants can access and use a private EOSIO network. The first security layer enforces TLS connections in the peer-to-peer network protocol; only those participants with a properly signed certificate are able to establish a TLS connection and communicate securely with other peers. The second security layer, which is optional, brings the concept of a *privacy group* or *security group*; if enabled, only those participants in the group are allowed to access data from the private network. Therefore, the first layer controls “connection” access while the second layer manages “data” access. Participants are considered logical entities with a designated EOSIO name and may host multiple nodes, potentially deployed in [Block Vault](../../03_plugins/blockvault_client_plugin/index.md) mode.
+
+## Concept
+
+Most businesses need to keep data private for regulatory compliance, customer privacy, or both. Since blockchain solutions have a history of public data and open architecture, businesses need certain privacy and security guarantees before they can adopt blockchain technology. The *Private Access Control* feature aims to bring EOSIO closer to the Enterprise by providing a general business security solution that can be used across a variety of [Use Cases](#use-cases).
+
+[[info | Access controls via consensus]]
+| Along with the inherent features of the EOSIO blockchain, the *Private Access Control* feature achieves security, robustness, and coordination through EOSIO consensus. The protocol feature can be enabled and participants added or removed from a *security group* only after the block that contains the corresponding transaction is approved by a supermajority of producer participants and the block becomes irreversible.
```

Comment:
Maybe link the "irreversible" text to the glossary? 
https://developers.eos.io/welcome/latest/glossary/index/#irreversible-block